src/main/java/lotto/view/LottoInputView.java

```diff
@@ -30,6 +35,22 @@ public static int inputBonusNumber() {
 		return input.nextInt();
 	}
 
+	public static int inputManualLottoCount() {
+		System.out.println(INPUT_MANUAL_LOTTO_COUNT_MESSAGE);
+		return input.nextInt();
+	}
```

Comment:
```suggestion
	public static int inputManualLottoCount() {
		System.out.println(INPUT_MANUAL_LOTTO_COUNT_MESSAGE);
		return Integer.parseInt(input.nextline());
	}
```

이처럼 숫자하나 입력후 enter 치는 곳에서는 nextInt 대신 
Integer.parseInt(scanner.nextLine()) 하시면 됩니다 

바로 parseInt를 해도 오류가 나지 않는 이유는 스캐너에서 nextLine 할 때 뒤에 개행문자는 제외하고 입력받기 때문이에요!

Reply:
아.. 이제 이해했어요! 친절한 답변 감사합니다!!